examples/a-valid-isin/profile.json

```diff
@@ -4,6 +4,10 @@
         { "name": "isin" }
     ],
     "rules": [
-		{ "field": "isin", "is": "aValid", "value": "ISIN" }
+		{ "constraints": 
```

Comment:
I don't believe this is a valid format for `rules:`
as far as I am aware rules has to contain a list of `rule` objects which then contain `constraint` objects

Reply:
Rules are objects that have  the following properties:
- optionally: "rule" for the rule name
- required: "constraints": an array of constraint objects.

Have i misunderstood your comment?

Reply:
where is this from?
in Profiles.md there is the statement:
`Every rule is a named collection of constraints, which can be any of:`

Reply:
found it (Schema.md)
Which is correct ?